ansible/app_options_files/empty

```diff
@@ -0,0 +1 @@
+
```

Comment:
Fair question. :) When you want to deploy a new basecoin infrastructure, you can add different sets of wallets up front. The config file stores them under app_options as a dict. Ansible has an input parameter that chooses the file from the app_options_files directory for the wallets during configuration. If there's no file, ansible errors out. Hence if you want ansible to create an empty testnet (with no wallets) you still have to have a file in this folder.
I'm considering refacting this completely, because later it turned out that the ethermint app_options part is not compatible with basecoin. This means that the content of this folder can only be used for basecoin and not much else...